content/applications/general/iot/connect.rst

```diff
@@ -0,0 +1,153 @@
+=============================
+IoT system connection to Odoo
+=============================
+
+Prerequisites
+=============
+
+To connect the IoT system to an Odoo database, the following prerequisites must be met:
+
+- The Odoo database must be a **production** instance.
+- The IoT app must be installed.
+- The IoT system must be connected to the network.
+- The computer connecting to Odoo must be on the same network as the IoT system.
+
+.. seealso::
+   - :doc:`iot_box`
+   - :doc:`windows_iot`
```

Comment:
change the indentation if this part is related to previous list last bullet point ?